test/Microsoft.SqlTools.ServiceLayer.PerfTests/Tests/Common.cs

```diff
@@ -12,7 +12,7 @@
 using Microsoft.SqlTools.ServiceLayer.Workspace.Contracts;
 using Xunit;
 
-namespace Microsoft.SqlTools.ServiceLayer.PerfTests.Tests
+namespace Microsoft.SqlTools.ServiceLayer.PerfTests
```

Comment:
This combo of changes to remove the .Tests from the namespaces and add .Tests to the default namespace doesn't seem right.